src/partials/header-content.hbs

```diff
@@ -63,10 +63,10 @@
 
             <div class="dropdown-label">TigerGraph Cloud</div>
             <div class="dropdown-category">
-              <a class="category-item" href="{{relativize site.components.cloud.url}}">{{site.components.cloud.title}}</a>
-              <a class="category-item" href="https://docs.tigergraph.com/cloudbeta/overview/">{{site.components.cloudBeta.title}}
+              <a class="category-item" href="{{relativize site.components.cloud4.url}}">{{site.components.cloud4.title}}
```

Comment:
@samzhou2 Did you test this? I don't think this works, because cloud4 is a module, not a component.

Reply:
@samzhou2 I guess Antora can treat a module as a component in this context, because cloudBeta (or Cloud 4.0) does shown up in the left nav menu, which now just uses an #each iterator to list every component.